packages/oui-file/file.less

```diff
@@ -224,6 +224,9 @@
     padding: 0;
     list-style: none;
     z-index: 0;
+    max-height: @oui-file-attachment-max-height;
+    overflow-y: auto;
+    box-shadow: 0px 0px 6px fade(@oui-color-info-light; 20%);
```

Comment:
For easing the migration to v3, can you set this in a  variable too ? (eg. https://github.com/ovh-ux/ovh-ui-kit/pull/410, https://github.com/ovh-ux/ovh-ui-kit/pull/420)

Reply:
@AxelPeter for sure ! It is done, I'll replicate this on v3

Reply:
Thanks :)